csFastFloat/csFastFloat.xml

```diff
@@ -0,0 +1,524 @@
+<?xml version="1.0"?>
```

Comment:
I consider this file as build-artifact, so it doesn't need to be checked in.

Not the scope for this PR, but as idea:
With https://dotnet.github.io/docfx you can create a static documentation site, which could be hosted here in the GH-pages to this project.
Additional markdown files, etc. can also be incorporated into that doc-build.
It's quite easy to automate and push the changes to GH-pages by CI.

Reply:
i'll make that an Issue, really intersting idea thanks for sharing!